easypost/beta/user.py

```diff
@@ -45,7 +45,7 @@ def get_next_page_of_children(
 
         data, api_key = requestor.request(method=RequestMethod.GET, url=url, params=params, beta=True)
         next_children_array: List[Any] = data.get("children", [])
-        if len(next_children_array) == 0 or not data.get("has_more", False):
+        if len(next_children_array) == 0:
```

Comment:
question: should this have just been an `and` instead of an `or`?

Reply:
+1